tests/util.rs

```diff
@@ -19,6 +19,10 @@ pub struct Wallet {
 }
 
 impl Wallet {
+    /// Select inputs to pay for `amount` of `asset`.
+    ///
+    /// The corresponding UTXOs are generated ad hoc and subsequently
+    /// stored in the internal state of the wallet.
     pub fn coin_select(&mut self, amount: Amount, asset: AssetId) -> Result<Vec<Input>> {
```

Comment:
Maybe this should be called `mint` instead then? :)

Reply:
I'm not sure about this one. 

I think the fact that the coins didn't exist before the call is just an implementation detail and we really just care about selecting coins when calling this.

I only added the docstring because a previous version of this change introduced some other arguments which made the whole test API worth explaining. Without that I think the API doesn't need to be documented, but it can't hurt.